unison-cli/src/Unison/CommandLine/InputPatterns.hs

```diff
@@ -448,7 +443,7 @@ find' cmd fscope =
     cmd
     []
     I.Visible
-    [(ZeroPlus, fuzzyDefinitionQueryArg)]
+    [(ZeroPlus, exactDefinitionArg)]
```

Comment:
Previously 'fuzzy' completers are now prefix-completers for performance reasons.